src/Evolution/Systems/Cce/Actions/BoundaryComputeAndSendToEvolution.hpp

```diff
@@ -72,5 +108,87 @@ struct BoundaryComputeAndSendToEvolution {
         true);
   }
 };
+
+/*!
+ * \ingroup ActionsGroup
+ * \brief Submits a request for CCE boundary data at the specified `time` to the
+ * `Cce::GhWorldtubeInterfaceManager`, and sends to the `EvolutionComponent`
+ * (template argument) it is ready.
+ *
+ * \details This uses the `Cce::GhWorldtubeInterfaceManager` to perform all of
+ * the work of managing the buffer of data sent from the Gh system and
+ * interpolating if necessary and supported. This dispatches then to
+ * `Cce::Actions::SendToEvolution<GhWorldtubeBoundary<Metavariables>,
+ * EvolutionComponent>` if the boundary data is ready, otherwise
+ * simply submitting the request and waiting for data to become available via
```

Comment:
use same tensing of verbs in both clauses (e.g. submits ... and waits. to match dispatches)